README.md

```diff
@@ -8,41 +8,35 @@ This project serves as the inventory source for the [Ansible Tower Configuration
 
 ## Usage 
 
-The projects `SAMPLE_CREDENTIAL_CONFIG.yml` file contains a list of variables which need to be changed to suit your own environment. Once these variables have been changed, the `bootstrap.yml` playbook is used to consume this configuration file,encrypt any variables containing sensitive information using the supplied Ansible vault password, and then place each of the variables into the relevant group_vars file.
+The projects `CREDENTIAL_CONFIG.yml` file contains a list of variables which need to be changed to suit your own tower environment. Once these variables have been changed,the `bootstrap.yml` playbook consumes the `CREDENTIAL_CONFIG.yml` file, encrypts any variables containing sensitive information using the supplied Ansible vault password, and then places each of the variables into the relevant group_vars file. The `CREDENTIAL_CONFIG.yml` file itself is then encrypted using the same Ansible vault password.
 
 ## Setup
 
 **NOTE**: This project should be forked and made private before proceeding with the following setup steps.
 
-1. Clone this project
+1. Clone this project.
 
 ```bash
 git clone https://github.com/<forked_repository_org>/tower_dummy_credentials
 ```
 
-2. Copy and rename the `SAMPLE_CREDENTIAL_CONFIG.yml` file locally. As the local copy of the config will sensitive information in plaintext, it should only be stored and referenced locally, and caution taken to not check it into a public repository.
+1. Change the variable values in the `CREDENTIALS_CONFIG.yml` file. A list of all variables that need to be changed along with their usage can be found [HERE](VARIABLES.md).
```

Comment:
Typo here, file is named CREDENTIAL_CONFIG.yml rather than CREDENTIALS_CONFIG.yml